doc/articles/dummies1.md

```diff
@@ -0,0 +1,143 @@
+# How to migrate existing UWP code to Uno
+
+There are two separate paths for using an existing UWP codebase on top of Uno:
+- An existing UWP application
+- An existing UWP library
+
+For migrating a UWP library, see [this article](howto-migrate-existing-code.md).
+
+# How to migrate UWP app to Uno?
+# Step by step guide.
+
+This guide assumes that you have some UWP app, and want to convert this app to be multiplatform.
+
+
+
+In short:
+* convert project into Uno,
+* find if something you use is not implemented in Uno,
+* if yes, you have two options:
+	* try to find workaround for this, or
+	* extend Uno
+* try to run converted app, and test it
+
+
+## create Uno project
```

Comment:
"Uno heads" is slang-term, and should be explained (I still doesn't know what _exactly_ 'Uno head' is...)

Reply:
Good point! Is 'Add Uno head projects for each platform' more clear?

The 'head project' is just the platform-specific project for each platform, eg the 'Android head', the 'iOS head' etc. I think we borrowed the term from Xamarin Forms/Xamarin Classic, I don't know where it originally comes from.

Reply:
'create uno project' is something natural: solution explorer, right click, add new, project, select Uno template.
'add head' is not in Solution Explorer menu :)

Maybe: "create uno project ... it this process, four projects ('heads') would be created, one for each platform (.Droid, .iOS, .UWP, .WASM), and one shared folder (.Shared)"?
.shared is not a project, and I consider it not a 'head'; am I right?

Reply:
The shared project is a project (because that's what VS calls it), but not a 'head' (because it's not platform-specific).

Btw I realised I documented the project structure, including the 'head' terminology, here: https://platform.uno/docs/articles/uno-app-solution-structure.html

A link would be appropriate :)

Reply:
Should be 'Create Uno projects' (capitalized)